x-pack/plugins/reporting/server/lib/store/ilm_policy_manager/ilm_policy_manager.ts

```diff
@@ -33,13 +43,95 @@ export class IlmPolicyManager {
     }
   }
 
+  /**
+   * This method is automatically called on the Stack Management > Reporting page, by the `` API for users with
+   * privilege to manage ILM, to notify them when attention is needed to update the policy for any reason.
+   */
+  public async checkIlmMigrationStatus(): Promise<IlmPolicyMigrationStatus> {
```

Comment:
Relocated this code from `x-pack/plugins/reporting/server/lib/deprecations/check_ilm_migration_status.ts`